### DISCUSSION TITLE
Coinbase account actually is mine but their used browser used Coinbase doing all this noun u manage to connect all browsers access to my account back because all their was used my belongings

### DISCUSSION THREAD

**Discussion by Agilan989** [community/community#106476] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

All the connections with coin base connect to my account back because all used my pic and my passkey to log in now u used same account thir used connect to my account back

**Comment by MrSchoool** [comment]
It appears that you are experiencing issues related to your Coinbase account and unauthorized access. To address this situation, follow these general steps:

1. Change Your Password:
Immediately change your password on Coinbase to secure your account. Choose a strong and unique password that includes a mix of uppercase and lowercase letters, numbers, and special characters.

2. Enable Two-Factor Authentication (2FA):
Enable two-factor authentication on your Coinbase account if you haven't already. This adds an extra layer of security by requiring a second form of verification in addition to your password.

3. Contact Coinbase Support:
Reach out to Coinbase's customer support as soon as possible. Report the unauthorized access, provide details about the incident, and ask for assistance in securing your account. Coinbase should have a support page or contact options on their website.

4. Check Connected Devices:
Review and manage the devices connected to your Coinbase account. Disconnect any unfamiliar devices and only authorize access from trusted devices.

5. Review Account Activity:
Check your account activity on Coinbase for any unauthorized transactions. If you notice any suspicious activity, report it to Coinbase support.

6. Monitor Other Accounts:
If you used the same password for other accounts, consider changing those passwords as well. It's good practice to use unique passwords for each of your accounts.

7. Use Email Verification:
Ensure that your email associated with Coinbase is secure. Consider enabling email verification for any login attempts or account changes.

8. Educate Yourself:
Be cautious of phishing attempts. Ensure that you are logging in only on the official Coinbase website. Avoid clicking on links or providing information in response to unsolicited messages.

Remember that these are general suggestions, and the specific steps may vary based on Coinbase's policies and procedures. If you are unable to resolve the issue through the suggested steps, contacting Coinbase support directly is crucial for resolving account security issues.

**Reply by queenofcorgis** [reply]
@MrSchoool 
We’ve clarified our stance on using generative AI tools like ChatGPT within our Community via [this announcement](https://github.com/orgs/community/discussions/69006). Please review the guidelines to ensure your post meets them as failure to adhere to those rules can result in action taken by our moderator team.  You can read our updated [Code of Conduct](https://github.com/community/community/blob/baa5cd9020ef3f2bbf86dca1191d9c90f9b0822d/CODE_OF_CONDUCT.md#reasonable-use-of-ai-generated-content) and the [announcement](https://github.com/orgs/community/discussions/69006) for more details. Thank you for helping us maintain an authentic and beneficial space for everyone.

**Comment by queenofcorgis** [comment]
Hi @Agilan989 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Coinbase. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
